### ISSUE TITLE
Gitbhub not working on latest version of Microsoft Edge (Chromium based)

### ISSUE THREAD

**Issue by jonatanelgstrom** [photopea/photopea#4229]
Hi,

I've been using Photopea as my main Photo editing ''software'' on Firefox for the longest time.
I recently switched fully to Microsoft Edge after a while of testing it to see how it stacks up to other modern browsers, and I found that when trying to use Photopea it just won't work.
It gives me an error message saying;
 ``Your broser is too old (no WebAssembly). Please, update it.´´
 
 Now, I know that Edge used to not have WebAssembly, however, this newer Chromium based one does, and I have it active (I have double-checked and verified this). It is also based on the same codebase as Google Chrome (the hint is in the name and logo).
 Google Chrome works fine with Photopea, so why doesn't Microsoft Edge work?

**Comment by photopea** [comment]
Hi, you probably configured your browser to disable WebAssembly. WebAssembly and Javascript support is required for running Photopea in a web browser.

Maybe this can help you: https://github.com/photopea/photopea/issues/3728

**Comment by albiom** [comment]
I literally just said that I checked that it was active...

**Comment by photopea** [comment]
I think you should deactivate that option, wich disables WebAssembly. There are hundreds of thousands of people using Photopea in Edge, and I tired it myself, too, so I am sure it works.

**Comment by albiom** [comment]
> I think you should deactivate that option, wich disables WebAssembly. There are hundreds of thousands of people using Photopea in Edge, and I tired it myself, too, so I am sure it works.

Hi,
I'm back again because I really want to use photopea because you've done really well with it. However, I still can't get it to work. I have enabled all option that have WebAssembly in the name (see attached picture) and there's no option for deactivating an option that disables WebAssembly.

Could you possibly provide a screenshot of said option, because I can't find it. 
Thanks in advance

The screenshot:
[options](https://imgur.com/a/BwDhSxd)

**Comment by photopea** [comment]
Could you uninstall the Edge completely, and then, install it again with the default options? I am sure Photopea works perfectly in a default configuration of every web browser. 

@celphogeek or maybe you could help us here?

**Comment by albiom** [comment]
> 

Thanks, I tried uninstalling it without saving configurations and then installing again still nothing

I think it's worth noting that I'm running the Windows 11 version of the browser

**Comment by photopea** [comment]
So the message is still "no WebAssembly"? What exactly version of Edge are you using?

**Comment by albiom** [comment]
> 

Yes, [this is the message](https://imgur.com/a/UpxjW2X) that pops up to be exact.
My version of Edge is 98.0.1108.62

**Comment by photopea** [comment]
Could you go to edge://settings , choose "Privacy, search and services" on the left side, scroll down to "Enhance your security on the web" and disable the switch?

![image (3)](https://user-images.githubusercontent.com/15955558/156672749-44cdd6b2-6a80-4f3d-82b8-8ccf63d9f52a.png)

**Comment by photopea** [comment]
For some reason, Microsoft thinks disabling WebAssembly improves the security, which is an absolute lie. No other browser disables WebAssembly.

**Comment by albiom** [comment]
> 

That was it, thank you!

**Comment by crazy-matt** [comment]
> 

Not entirely sure of what is being stated here. This type of comment does not help in making the web a safer place. What about code injection? I've seen somewhere cases of WebAssembly being leveraged to takeover on a machine and start some crypto mining. 
I prefer to have a browser with WebAssembly disabled for my day-to-day browsing and switch to a 2nd one when WebAssembly is really needed.

**Comment by photopea** [comment]
@crazy-matt Any website can use your device to mine cryptocurrencies with Javascript, there is no need for WebAssembly. 

Mining cryptocurrencies, or doing any other kind of computation, is not harmful to you or your device. It only uses your electricity / drains your battery.

**Comment by photopea** [comment]
@crazy-matt Any website can use your device to mine cryptocurrencies with Javascript, there is no need for WebAssembly. 

Mining cryptocurrencies, or doing any other kind of computation, is not harmful to you or your device. It only uses your electricity / drains your battery. I have no idea what you mean by "takeover on a machine".

**Comment by photopea** [comment]
@crazy-matt Any website can use your device to mine cryptocurrencies with Javascript, there is no need for WebAssembly.

Mining cryptocurrencies, or doing any other kind of computation, is not harmful to you or your device. It only uses your electricity / drains your battery. 

I have no idea what you mean by "takeover on a machine". You can always close that website.
